### DISCUSSION TITLE
Oracle feature

### DISCUSSION THREAD

**Discussion by rifkyaziz** [ssantoshp/Empyrial#30] [General]
hello ssantoshp,
After updating empyrial to the latest version, i notice that oracle module has been deleted. Can i know whether oracle is discontinued or still in development?
Thanks

**Comment by santoshlite** [comment]
Hi @rifkyaziz,

Indeed, Oracle has been removed from Empyrial because I thought that users were not making much use of this and because I wanted to focus on the backtesting part of Empyrial. Funny that you're the 3rd person asking me that, I'll maybe consider making a module for oracle only, what do you think?

Best ;)

**Reply by rifkyaziz** [reply]
@ssantoshp I definitely will love that. And thanks for your great work, this package was really helpful for me

**Reply by santoshlite** [reply]
Thanks for the feedback 😄 I'll definitely try to work on that!

**Reply by santoshlite** [reply]
@rifkyaziz Hey, took a long time but here it is: https://github.com/ssantoshp/The-Oracle

The Oracle is back!
